modules/backend/formwidgets/richeditor/assets/js/richeditor.js

```diff
@@ -95,46 +95,57 @@
         froalaOptions.imageStyles = this.options.imageStyles
             ? this.options.imageStyles
             : {
-              'oc-img-rounded': 'Rounded',
-              'oc-img-bordered': 'Bordered'
+                'oc-img-rounded': 'Rounded',
+                'oc-img-bordered': 'Bordered'
             }
 
         froalaOptions.linkStyles = this.options.linkStyles
             ? this.options.linkStyles
             : {
-              'oc-link-green': 'Green',
-              'oc-link-strong': 'Thick'
+                'oc-link-green': 'Green',
+                'oc-link-strong': 'Thick'
             }
 
         froalaOptions.paragraphStyles = this.options.paragraphStyles
             ? this.options.paragraphStyles
             : {
-              'oc-text-gray': 'Gray',
-              'oc-text-bordered': 'Bordered',
-              'oc-text-spaced': 'Spaced',
-              'oc-text-uppercase': 'Uppercase'
+                'oc-text-gray': 'Gray',
+                'oc-text-bordered': 'Bordered',
+                'oc-text-spaced': 'Spaced',
+                'oc-text-uppercase': 'Uppercase'
             }
 
         froalaOptions.tableStyles = this.options.tableStyles
             ? this.options.tableStyles
             : {
-              'oc-dashed-borders': 'Dashed Borders',
-              'oc-alternate-rows': 'Alternate Rows'
+                'oc-dashed-borders': 'Dashed Borders',
+                'oc-alternate-rows': 'Alternate Rows'
             }
 
         froalaOptions.tableCellStyles = this.options.tableCellStyles
             ? this.options.tableCellStyles
             : {
-              'oc-cell-highlighted': 'Highlighted',
-              'oc-cell-thick-border': 'Thick'
+                'oc-cell-highlighted': 'Highlighted',
+                'oc-cell-thick-border': 'Thick'
             }
 
         froalaOptions.toolbarButtonsMD = froalaOptions.toolbarButtons
         froalaOptions.toolbarButtonsSM = froalaOptions.toolbarButtons
         froalaOptions.toolbarButtonsXS = froalaOptions.toolbarButtons
 
-        if (this.options.htmlAllowedEmptyTags) {
-            froalaOptions.allowEmptyTags = this.options.htmlAllowedEmptyTags.split(/[\s,]+/)
+        if (this.options.allowEmptyTags) {
+            // Filter the tags list to split eventual tag.class and add them if necessary
+            let filteredTagList = [];
+            this.options.allowEmptyTags.split(/[\s,]+/).forEach(tagString => {
```

Comment:
is .forEach supported in IE11?

Reply:
@LukeTowers according to MDN, it is: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Browser_compatibility

Reply:
But `let` isn't properly supported.